__tests__/app.test.js

```diff
@@ -69,3 +69,34 @@ describe("/api", () => {
     });
   });
 });
+
+describe("PATCH /api/articles/:article_id ", () => {
+  test("201: returns an object with updated votes property by the provided number", () => {
+    const req = { inc_votes: 3 };
+    const expected = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: expect.any(Date),
+      votes: 103,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(req)
+      .expect(201)
+      .then(({ body }) => {
+        const formattedResponse = convertTimestampToDate(body.updatedVotes);
```

Comment:
This would be more efficient true to what you're testing if you changed the "created_at" on line 82 to expect.any(String).

Reply:
I thought it suppose to be a date. Expecting string makes it a different type of data. That is why I have used convertTimeStampToDate. The same is on line 49. Please let me know your thoughts.